errata_tool/tests/conftest.py

```diff
@@ -160,9 +160,18 @@ def build(monkeypatch, mock_get):
 
 
 @pytest.fixture
-def variant(monkeypatch, mock_get):
+def rhceph_variant(monkeypatch, mock_get):
```

Comment:
This is fine, but would you mind separating this rename into a separate Git commit so it's easier to review?

Reply:
Like, "rename variant fixture"

Reply:
Will do.